pages/architecture.md

```diff
@@ -33,6 +33,61 @@ Here's a diagram describing this flow:
 To see if an instrumentation library already exists for your platform, see the
 list of [existing instrumentations]({{ site.github.url}}/pages/existing_instrumentations).
 
+Example flow
+-----------------------
+
+As mentioned in the overview, identifiers are sent in-band and details are sent
+out-of-band to Zipkin. In both cases, trace instrumentation is responsible for
+creating valid traces and rendering them properly. For example, a tracer ensures
+parity between the data it sends in-band (downstream) and out-of-band (async to
+Zipkin).
+
+Here's an example sequence of http tracing. Notice timing data is reported to
+Zipkin after the application response is completed.
```

Comment:
"after the application response" may also be accurate here, but I believe the diagram is trying to show the tracing of the HTTP call `GET /foo` being made by "Application". Maybe it would be less confusing/more accurate to say "after the HTTP call is completed."